admin-tools/build_and_check_manifest.py

```diff
@@ -10,15 +10,8 @@ def generate_avaliable_builtins_names():
     for module in modules:
         vars = dir(module)
         for name in vars:
-            var = getattr(module, name)
-            if (
-                hasattr(var, "__module__")
-                and var.__module__.startswith("mathics.builtin.")
-                and var.__module__ != "mathics.builtin.base"
-                and is_builtin(var)
-                and not name.startswith("_")
-                and var.__module__ == module.__name__
-            ):  # nopep8
+            var = contributing_builtin_var(module, name)
+            if var:  # nopep8
```

Comment:
When I look at the function signature for this: 

```python
def contributing_builtin_var(module, name) -> Optional[type]:
```

I don't see a reason not to use `if var is not None`  Also the signature is incorrect, but I will note that below